app/src/main/kotlin/com/wire/android/ui/calling/controlButtons/CameraButton.kt

```diff
@@ -10,22 +14,41 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import com.wire.android.R
 import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.util.extension.checkPermission
 
 @Composable
 fun CameraButton(
     isCameraOn: Boolean = false,
-    onHangUpButtonClicked: () -> Unit
+    cameraPermissionDenied: () -> Unit,
```

Comment:
```suggestion
    onCameraPermissionDenied: () -> Unit,
```

What do you think of renaming to the above to follow the pattern we have on other permission arguments ?

Reply:
Great catch!